<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>500</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
border-radius:15px;
</string>
    </property>
    <widget class="QWidget" name="widgetLb" native="true">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>370</y>
       <width>240</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
border-radius:10px;</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>31</width>
            <height>21</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>31</width>
            <height>21</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image: url(:/yes.png);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="styleSheet">
           <string notr="true">font: 700 15pt &quot;Microsoft YaHei UI&quot;;
color: rgb(25, 254, 205);

</string>
          </property>
          <property name="text">
           <string>认证成功</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="headpicLb">
     <property name="geometry">
      <rect>
       <x>117</x>
       <y>40</y>
       <width>266</width>
       <height>266</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/crc.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="videoLb">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>500</width>
       <height>500</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="widgetLb_2" native="true">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>370</y>
       <width>301</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
border-radius:10px;</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>31</width>
            <height>21</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>31</width>
            <height>21</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image: url(:/yes.png);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="styleSheet">
           <string notr="true">font: 700 15pt &quot;Microsoft YaHei UI&quot;;
color: rgb(255, 255, 255);
color: rgb(255, 39, 10);
</string>
          </property>
          <property name="text">
           <string>认证成功，晚归</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>500</y>
       <width>500</width>
       <height>100</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(112, 112, 112);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="fault_Lb">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>378</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <bold>true</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>识别失败</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <zorder>videoLb</zorder>
    <zorder>widgetLb</zorder>
    <zorder>headpicLb</zorder>
    <zorder>widgetLb_2</zorder>
    <zorder>label_7</zorder>
    <zorder>fault_Lb</zorder>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>0</y>
      <width>450</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
border-radius:10px;
background-color: rgb(112, 112, 112);
background-color: rgb(4, 131, 212);</string>
    </property>
    <widget class="QLabel" name="headLb">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>145</y>
       <width>150</width>
       <height>150</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/face.png);
background-color: rgb(180, 180, 180);

border-radius:75px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>450</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 21pt &quot;Microsoft YaHei UI&quot;;
color: rgb(255, 255, 255);
font-weight: bold;</string>
     </property>
     <property name="text">
      <string>宿舍人脸识别门禁系统</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>310</y>
       <width>451</width>
       <height>262</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,0">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 20pt &quot;Microsoft YaHei UI&quot;;
color: rgb(138, 138, 138);
color: rgb(180, 180, 180);

background-color: rgba(255, 0, 0, 0);</string>
          </property>
          <property name="text">
           <string>姓名：   </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 20pt &quot;Microsoft YaHei UI&quot;;
color: rgb(138, 138, 138);
background-color: rgba(255, 0, 0, 0);
color: rgb(180, 180, 180);</string>
          </property>
          <property name="text">
           <string>学号：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 20pt &quot;Microsoft YaHei UI&quot;;
color: rgb(138, 138, 138);
background-color: rgba(255, 0, 0, 0);
color: rgb(180, 180, 180);</string>
          </property>
          <property name="text">
           <string>学院：   </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_14">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 20pt &quot;Microsoft YaHei UI&quot;;
color: rgb(138, 138, 138);
background-color: rgba(255, 0, 0, 0);
color: rgb(180, 180, 180);</string>
          </property>
          <property name="text">
           <string>宿舍号：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_15">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 20pt &quot;Microsoft YaHei UI&quot;;
color: rgb(138, 138, 138);
background-color: rgba(255, 0, 0, 0);
color: rgb(180, 180, 180);</string>
          </property>
          <property name="text">
           <string>刷脸时间：</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLineEdit" name="nameEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="tabletTracking">
           <bool>true</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 20pt &quot;Microsoft YaHei UI&quot;;
background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);
border:none;
border: 1px solid white; border-radius: 10px;</string>
          </property>
          <property name="text">
           <string>0001</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="numberEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="tabletTracking">
           <bool>true</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 20pt &quot;Microsoft YaHei UI&quot;;
background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);
border:none;
border: 1px solid white; border-radius: 10px;</string>
          </property>
          <property name="text">
           <string>0001</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="schoolEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="tabletTracking">
           <bool>true</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">
font: 20pt &quot;Microsoft YaHei UI&quot;;
background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);
border:none;
border: 1px solid white; border-radius: 10px;</string>
          </property>
          <property name="text">
           <string>0001</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="dormitory_number_Edit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="tabletTracking">
           <bool>true</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 20pt &quot;Microsoft YaHei UI&quot;;
background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);
border:none;
border: 1px solid white; border-radius: 10px;</string>
          </property>
          <property name="text">
           <string>0001</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="timeEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="tabletTracking">
           <bool>true</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 20pt &quot;Microsoft YaHei UI&quot;;
background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);
border:none;
border: 1px solid white; border-radius: 10px;</string>
          </property>
          <property name="text">
           <string>0001</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>250</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/icon.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
